src/test/java/spoon/reflect/ast/CloneTest.java

```diff
@@ -283,19 +283,13 @@ public void testCloneKeepsImplicitModifierState() {
 		CtClass<?> clonedTest = test.clone();
 
 		// Sanity checks
-		assertEquals(
-				"Class has the wrong amount of modifiers",
-				2,
-				test.getExtendedModifiers().size()
+		assertEquals(2, test.getExtendedModifiers().size(), "Class has the wrong amount of modifiers"
 		);
 		assertModifierImplicitness(test.getExtendedModifiers(), ModifierKind.PUBLIC, true);
 		assertModifierImplicitness(test.getExtendedModifiers(), ModifierKind.ABSTRACT, false);
 
 		// Test that the implicit state was kept
-		assertEquals(
-				"Clone has wrong amount of modifiers",
-				2,
-				clonedTest.getExtendedModifiers().size()
+		assertEquals(2, clonedTest.getExtendedModifiers().size(), "Clone has wrong amount of modifiers"
```

Comment:
Could you manually pull up the `);` from the next line to this one? Looks very odd right now.